### DISCUSSION TITLE
Add sub-domain to panopticon

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#591] [Q&A (Community Support)]
Hi, I need help with adding a sub-domain to Panopticon.
Sub-domain is the stagging of the root domain. Both pages are upgraded to Joomla 5.
Root is domain.com and subdomain is domain.com/upgrade
The root domain has been added and is working as expected.
When add a subdomain get a message
![Akeeba-Panopticon](https://github.com/akeeba/panopticon/assets/17553018/66675912-5af1-40e5-86e1-9ab7c35313b4)
I tried to reinstall core Joomla files on both sites and regenerate .htaccess on both sites, but had no luck.
Also, Panopticon has been reinstalled.
On any of these pages, don't use admintools, and htaccess is the Joomla default.
All files and folders inside the tmp folder of Akeeba Panopticon are deleted.
After enabling Debug System error is not changed.
Any ideas?

**Comment by nikosdion** [comment]
Do not put the subdomain's root under your main site's root. SEF URLs, which are necessary for the Joomla! API application to work correctly, will be messed up.
